sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Common/AnalyzerAdditionalFile.cs

```diff
@@ -27,6 +27,10 @@ namespace SonarAnalyzer.Common
 {
     public sealed class AnalyzerAdditionalFile : AdditionalText
     {
+
+public string field1;

```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make this field 'private' and encapsulate it in a 'public' property. ([csharpsquid:S1104](https://sonarcloud.io/organizations/sonarsource/rules#rule_key=csharpsquid%3AS1104))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=sonaranalyzer-dotnet&pullRequest=1435&issues=AWO3CDQT4427HkKVZCdq&open=AWO3CDQT4427HkKVZCdq)

[ISSUE_KEY]: AWO3CDQT4427HkKVZCdq